spring-cloud-stream/src/test/resources/org/springframework/cloud/stream/binder/sink-binding-pubsub-test.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.stream.bindings.input=topic:testpubsub
+spring.cloud.stream.group=tgroup
```

Comment:
Well... consider a very general wiretap case. Perhaps a component is calculating some simple metrics on traffic across multiple streams, hence multiple "groups". This would be a macro version of what some users accomplish with global channel-interceptors in spring-integration.